pkgs/development/ocaml-modules/irmin/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, buildDunePackage, ocaml
```

Comment:
You don’t need `stdenv` as argument when you only use `stdenv.lib`: you can directly take `lib` as argument.